sql/core/src/main/scala/org/apache/spark/sql/internal/SharedState.scala

```diff
@@ -112,6 +113,11 @@ private[sql] class SharedState(
    */
   val cacheManager: CacheManager = new CacheManager
 
+  /**
+   * Class for caching streaming query instance.
+   */
+  val streamQueryStore: StreamQueryStore = new StreamQueryStore()
```

Comment:
Can we reuse the existing `activeStreamingQueries` at line 125?
If we gets all `StreamingQueryManager`, it seems that you can get all active queries by `StreamingQueryManager.activeQueries` invocation.

cc @brkyvz since he added `activeStreamingQueries` recently.

Reply:
ok

Reply:
We can not reuse `activeStreamingQueries ` to get all active/inactive stream queries, as it removed terminated stream. Perhaps, we should create a new class to serve this.

Reply:
Move the `activeStreamingQueries` into new added `StreamQueryStore` class